tools/psa/generate_mbed_spm_partition_code.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/python
+# Copyright (c) 2017-2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+import os
+from os.path import join as path_join
+from jinja2 import Environment, FileSystemLoader, StrictUndefined
+
+from mbed_spm_tfm_common import Manifest, validate_partition_manifests, manifests_discovery
```

Comment:
Needs to be a relative import:
```suggestion
from .mbed_spm_tfm_common import Manifest, validate_partition_manifests, manifests_discovery
```